src/main/java/net/dv8tion/jda/api/entities/MessageChannel.java

```diff
@@ -2174,10 +2163,7 @@ default MessagePaginationAction getIterableHistory()
      *         The emote to remove
      *
      * @throws java.lang.IllegalArgumentException
-     *         <ul>
-     *             <li>If provided {@code messageId} is {@code null} or empty.</li>
-     *             <li>If provided {@code emote} is {@code null}.</li>
-     *         </ul>
+     *         If provided {@code emote} is {@code null}.
```

Comment:
Why did this lose the bullet point about the `messageId`?

Reply:
Same as above.